package.json

```diff
@@ -87,11 +87,49 @@
+    "test-docker": "docker-compose run sequelize /bin/sh -c \"npm run all\"",
+    "test-docker-unit": "docker-compose run sequelize /bin/sh -c \"npm run test-unit-all\"",
     "build-docker": "docker-compose build",
-    "docs": "node docs/docs-generator.js"
+    "docs": "node docs/docs-generator.js",
+    "jshint": "./node_modules/.bin/jshint lib test",
+    "teaser": "echo ''; node -pe \"Array(20 + process.env.DIALECT.length + 3).join('#')\"; echo '# Running tests for '$DIALECT' #'; node -pe \"Array(20 + process.env.DIALECT.length + 3).join('#')\";echo '';",
+    "test-unit": "./node_modules/.bin/mocha --globals setImmediate,clearImmediate --ui tdd --check-leaks --colors -t 15000 --reporter spec 'test/unit/**/*.js'",
+    "test-unit-mysql":"DIALECT=mysql npm run test-unit",
+    "test-unit-postgres":"DIALECT=postgres npm run test-unit",
+    "test-unit-postgres-native":"DIALECT=postgres-native npm run test-unit",
+    "test-unit-mariadb":"DIALECT=mariadb npm run test-unit",
+    "test-unit-sqlite":"DIALECT=sqlite npm run test-unit",
+    "test-unit-mssql":"DIALECT=mssql npm run test-unit",
+    "test-unit-all":"npm run test-unit-mysql && npm run test-unit-postgres && npm run test-unit-postgres-native && npm run test-unit-mssql && npm run test-unit-sqlite && npm run test-unit-mariadb",
+    "test-integration": "./node_modules/.bin/mocha --globals setImmediate,clearImmediate --ui tdd --check-leaks --colors -t 15000 --reporter spec --grep ''$GREP'' 'test/integration/**/*.test.js'",
+    "test-integration-mysql":"DIALECT=mysql npm run test-integration",
+    "test-integration-postgres":"DIALECT=postgres npm run test-integration",
+    "test-integration-postgres-native":"DIALECT=postgres-native npm run test-integration",
+    "test-integration-mariadb":"DIALECT=mariadb npm run test-integration",
+    "test-integration-sqlite":"DIALECT=sqlite npm run test-integration",
+    "test-integration-mssql":"DIALECT=mssql npm run test-integration",
+    "test-integration-all":"npm run test-integration-mysql && npm run test-integration-postgres && npm run test-integration-postgres-native && npm run test-integration-mssql && npm run test-integration-sqlite && npm run test-integration-mariadb",
+    "cover": "rm -rf coverage && npm run teaser && COVERAGE=true ./node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha --report lcovonly -- -t 30000 --ui tdd 'test/integration/**/*.test.js'",
+    "mysql-cover": "DIALECT=mysql npm run cover && mv coverage coverage-mysql",
+    "sqlite-cover": "DIALECT=sqlite npm run cover && mv coverage coverage-sqlite",
+    "postgres-cover": "DIALECT=postgres npm run cover && mv coverage coverage-postgres",
+    "postgres-native-cover": "DIALECT=postgres-native npm run cover && mv coverage coverage-postgres-native",
+    "mariadb-cover": "DIALECT=mariadb npm run cover && mv coverage coverage-mariadb",
+    "mssql-cover": "DIALECT=mssql npm run cover && mv coverage coverage-mssql",
```

Comment:
lets invert these to `cover-mysql` so they have the same syntax as the test scripts